Gemfile

```diff
@@ -20,6 +20,9 @@ gem "hanami-webconsole", github: "hanami/webconsole", branch: "main"
 
 gem "hanami-devtools", github: "hanami/devtools", branch: "main"
 
+gem "dry-system", github: "alassek/dry-system", branch: 'provider-class-override'
+gem "dry-inflector", github: "dry-rb/dry-inflector", branch: "main"
```

Comment:
Is this dry-inflector line strictly needed for this PR?

Reply:
I removed the dry-inflector line here and everything still passes, so I've pushed that change up too.